pymeasure/instruments/eurotest/eurotestHPP120256.py

```diff
@@ -0,0 +1,320 @@
+        """
+        Waits until HV voltage output reaches the voltage ouput setting.
+        Checks the voltage output every check_period seconds and raises an exception
+        if the voltage output doesn't reach the voltage setting until the timeout time.
+        :param check_period: voltage output will be measured every check_period (seconds) time
+        :param timeout: time (seconds) give to the voltage output to reach the voltage setting
+        :return: None
+        :raises: Exception if the voltage output can't reach the voltage setting
+        before the timeout completes (seconds)
+        """
+        log.info("Executing the wait_for_voltage_output_set function.")
+
+        ref_time = time.time()
+        future_time = ref_time + timeout
+
+        voltage_output_setting = self.voltage[1]
+        time.sleep(self.COMMAND_DELAY)
+        error = 0.02  # error of +-ten volts (0.01kV) to enter into the voltage stable window
+        voltage_output = self.measure_voltage[1]
+        voltage_output_set = (voltage_output > (voltage_output_setting - error)) and \
+                             (voltage_output < (voltage_output_setting + error))
+
+        while not voltage_output_set:
+            actual_time = time.time()
+            log.info(f"\tWaiting for voltage output set. "
+                     f"Reading output voltage every {check_period} seconds.\n"
+                     f"\tTimeout: {timeout} seconds. Elapsed time: "
+                     f"{round(actual_time - ref_time, ndigits=1)} seconds.")
+
+            time.sleep(check_period)  # wait for voltage output reaches the voltage output setting
+            voltage_output = self.measure_voltage[1]
+            voltage_output_set = (voltage_output > (voltage_output_setting - error)) and \
+                                 (voltage_output < (voltage_output_setting + error))
+            log.debug("voltage_output_valid_range: "
+                      "[" + str(voltage_output_setting - error) +
+                      ", " + str(voltage_output_setting + error) + "]")
+            log.debug("voltage_output: " + str(voltage_output))
+            if actual_time > future_time:
+                raise Exception("Timeout for wait_for_voltage_output_set function")
+
+        # if you are here is because the voltage_setting
+        # has been reaches at the output of the HV Voltage source
+
+    # Constructor
+    def __init__(self, adapter, response_encoding="iso-8859-2", query_delay=0.1, **kwargs):
+        # This instrument use this encoding, so we have to take into account
+        self.response_encoding = response_encoding
+        # Delay in s to sleep between the write and read occuring in a query
+        self.query_delay = query_delay
+        # comment next line if you want to pass the test
+        # adapter.connection.encoding = self.response_encoding
+        # comment next line if you want to pass the test
+        # adapter.connection.query_delay = self.query_delay
```

Comment:
You commented them already. Which test do you want to pass?

Reply:
Code have changed. Now those have been removed. Thanks